<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Commonly used software</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 64px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h2 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h3 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h4 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h5 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h6 {
  padding-top: 69px;
  margin-top: -69px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Jaffe lab onboarding materials</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="basic_intro.html">Basic Intro</a>
</li>
<li>
  <a href="communication.html">Communication</a>
</li>
<li>
  <a href="setup.html">Setup</a>
</li>
<li>
  <a href="analysis.html">Analysis</a>
</li>
<li>
  <a href="meetings.html">Meetings</a>
</li>
<li>
  <a href="food.html">Eating nearby</a>
</li>
<li>
  <a href="baltimore.html">Baltimore</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="http://github.com/LieberInstitute">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Commonly used software</h1>

</div>


<div id="bioconductor" class="section level2">
<h2>Bioconductor</h2>
<ul>
<li><a href="http://lcolladotor.github.io/2014/10/16/startbioc/#.XXf1QZNKi50">Where do I start using Bioconductor?</a> blog post by Leo.</li>
<li><a href="http://lcolladotor.github.io/2017/03/06/how-to-ask-for-help-for-bioconductor-packages/#.XXf1YpNKi50">How to ask for help for Bioconductor packages</a> blog post by Leo.</li>
<li>Bioconductor <a href="https://bioconductor.org/packages/release/BiocViews.html#___Workflow">workflows</a> such as <a href="http://bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html">rnaseqGene</a> and <a href="http://bioconductor.org/packages/release/workflows/vignettes/sequencing/inst/doc/sequencing.html">sequencing</a> are very helpful to get started in the RNA sequencing (RNA-seq) field.</li>
<li>BioC2019 workshops are available <a href="https://rebrand.ly/biocworkshops2019">here</a>.</li>
<li><em><a href="https://bioconductor.org/packages/3.9/limma">limma</a></em>: is one of the main packages we use for analyzing data, particularly with the limma-voom method. Check the Bioconductor workflows that explain limma in more detail <a href="https://bioconductor.org/packages/release/BiocViews.html#___Workflow">here</a>.</li>
<li><em><a href="https://bioconductor.org/packages/3.9/GenomicRanges">GenomicRanges</a></em>: is the package we use for interacting with genomic data across ranges. Check the <a href="http://bioconductor.org/packages/release/bioc/vignettes/GenomicRanges/inst/doc/GenomicRangesIntroduction.html">introduction</a> for more information.</li>
<li><em><a href="https://bioconductor.org/packages/3.9/SummarizedExperiment">SummarizedExperiment</a></em>: is the package we use for creating and interacting with <code>RangedSummarizedExperiment</code> objects.</li>
<li><em><a href="https://bioconductor.org/packages/3.9/bsseq">bsseq</a></em>: is highly useful for analyzing DNA methylation data, particularly whole genome bisulfite sequencing (WGBS) data.</li>
<li><em><a href="https://bioconductor.org/packages/3.9/Biostrings">Biostrings</a></em>: is useful for dealing with sequence information such as the human genome sequence.</li>
<li><em><a href="https://bioconductor.org/packages/3.9/recount">recount</a></em>: contains the <code>getRPKM</code> function that we use frequently and allows accessing data from many human RNA-seq projects.</li>
<li><em><a href="https://bioconductor.org/packages/3.9/derfinder">derfinder</a></em>: contains functions for BigWig files and other related RNA-seq methods.</li>
</ul>
</div>
<div id="lab-software" class="section level2">
<h2>Lab software</h2>
<ul>
<li><em><a href="https://github.com/LieberInstitute/jaffelab">jaffelab</a></em> (<a href="http://lieberinstitute.github.io/jaffelab">documentation</a>): contains several functions we’ve written over the years that we find helpful for different analyses.</li>
<li><em><a href="https://github.com/LieberInstitute/sgejobs">sgejobs</a></em> (<a href="http://lieberinstitute.github.io/sgejobs">documentation</a>): contains helper functions for writing and interacting with SGE jobs at JHPCE.</li>
<li><a href="https://github.com/LieberInstitute/jhpce_mod_source">jhpce_mod_source</a> and <a href="https://github.com/LieberInstitute/jhpce_module_config">jhpce_module_config</a> are our GitHub repositories for our LIBD modules.</li>
<li><em><a href="https://github.com/LieberInstitute/libdRSE">libdRSE</a></em> (<a href="http://lieberinstitute.github.io/libdRSE">documentation</a>): is similar to <em><a href="https://bioconductor.org/packages/3.9/recount">recount</a></em> for accessing coverage data from LIBD projects.</li>
<li><em><a href="https://github.com/LieberInstitute/shinycsv">shinycsv</a></em>: allows interactively exploring a table.</li>
<li><em><a href="https://github.com/LieberInstitute/LIBDpheno">LIBDpheno</a></em>: contains phenotype information for LIBD samples.</li>
<li><em><a href="https://github.com/LieberInstitute/recount.bwtool">recount.bwtool</a></em>: is an R package for dealing with BigWig files using <code>bwtool</code>. Check also <em><a href="https://github.com/ChristopherWilks/bamcount">bamcount</a></em> by Christopher Wilks.</li>
<li><em><a href="https://github.com/LieberInstitute/brainflowprobes">brainflowprobes</a></em>: is an R package for designing flow probes written by <a href="https://www.linkedin.com/in/amanda-price-762a982a/">Amanda J Price</a>.</li>
<li>LIBD RNA-seq processing pipeline: <a href="https://github.com/LieberInstitute/RNAseq-pipeline">initial version</a> and <a href="https://github.com/LieberInstitute/RNAsp">nextflow version</a>.</li>
<li><a href="https://github.com/LieberInstitute/globus"><code>globus</code></a>: our Globus share endpoints such that others can access our data.</li>
<li><em><a href="https://github.com/ComunidadBioInfo/easyweb">easyweb</a></em> for easily creating your own personal R markdown website.</li>
</ul>
</div>
<div id="writing-papers" class="section level2">
<h2>Writing papers</h2>
<ul>
<li><a href="https://blog.f1000.com/2019/07/31/how-to-write-academic-documents-with-googledocs-using-f1000workspace/">How to write academic documents with GoogleDocs using F1000Workspace</a> blog post. Check the <a href="http://lcolladotor.github.io/2019/04/02/how-to-write-academic-documents-with-googledocs/#.XXfKcJNKi50">original</a> version for some additional information.</li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
